src/app/app.module.ts

```diff
@@ -77,11 +77,11 @@ import { PortalStatisticsComponent } from './components/portal-statistics/portal
         redirectUrl: window.location.origin,
         postLogoutRedirectUri: window.location.origin,
         clientId: environment.identityServerClientId,
-        scope: 'openid profile email', // Adjust the scopes as needed
+        scope: 'openid profile email offline_access',
         responseType: 'code',
         silentRenew: true,
         useRefreshToken: true,
-        renewTimeBeforeTokenExpiresInSeconds: 30,
+        logLevel: LogLevel.Debug,
```

Comment:
This only for testing purposes I think?

Reply:
True, but it is in the frontend, so we cannot make easily it as a environment variable. It's much faster to push.